pallets/handles/src/handles-utils/constants.rs

```diff
@@ -2,27 +2,88 @@
 
 use core::ops::RangeInclusive;
 
-/// Character that are allowed.
-pub const ALLOWED_UNICODE_CHARACTER_RANGES: [RangeInclusive<u16>; 21] = [
-	0x0020..=0x007F, // Basic Latin
-	0x0080..=0x00FF, // Latin-1 Supplement
-	0x0100..=0x017F, // Latin Extended-A
-	0x0370..=0x03FF, // Greek and Coptic
-	0x0400..=0x04FF, // Cyrillic
-	0x0500..=0x052F, // Cyrillic Supplementary
-	0x0590..=0x05FF, // Hebrew
-	0x0600..=0x06FF, // Arabic
-	0x0900..=0x097F, // Devanagari
-	0x0980..=0x09FF, // Bengali
-	0x0E00..=0x0E7F, // Thai
-	0x1100..=0x11FF, // Hangul Jamo
-	0x1E00..=0x1EFF, // Latin Extended Additional
-	0x1F00..=0x1FFF, // Greek Extended
-	0x3040..=0x309F, // Hiragana
-	0x30A0..=0x30FF, // Katakana
-	0x3400..=0x4DBF, // CJK Unified Ideographs Extension A
-	0x4E00..=0x9FFF, // CJK Unified Ideographs
-	0xAC00..=0xD7AF, // Hangul Syllables
-	0xF900..=0xFAFF, // CJK Compatibility Ideographs
-	0xFB50..=0xFDFF, // Arabic Presentation Forms-A
+/// Characters that are allowed.
+#[rustfmt::skip]
+pub const ALLOWED_UNICODE_CHARACTER_RANGES: [RangeInclusive<u16>; 81] = [
```

Comment:
We didn't do this before due to the short list, but do we need to have a `const fn` that converts this at build time to a compressed list of ranges? Aka (in decimal) [1-5, 5-7, 9-10]` -> `[1-7, 9-10]`?

This does turn into `O(<range count>*<handle char length>)` so likely worth it with the increased size and with many abutting ranges.

Could also just do it manually (which might be easier).

Reply:
Added a test that generates the compacted list, which is now down to 54 items.